src/test/integration/stories/deploy_from_invalid_file_test.js

```diff
@@ -16,14 +16,17 @@ import path from 'path';
 
 import DeployFromFilePageObject from '../deploy/deployfromfile_po';
 
+import remote from '../../../../node_modules/selenium-webdriver/remote';
```

Comment:
Remove newlines and try to sort it in alphabetic order. It's easier to read this way. Also you can just use

```javascript
import remote from 'selenium-webdriver/remote';
```